<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>LED</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>OPEN I2C 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>LED EXT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open_i2c2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>OPEN I2C 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_write_i2c2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>WRITE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_read_i2c2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>READ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditReg">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditVal">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Reg</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Value</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditReg2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditVal2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
